docs/pages/release_notes.md

```diff
@@ -28,13 +28,15 @@ This is a minor release.
 *   doc
     *   [#791](https://github.com/pmd/pmd/issues/791): \[doc] Documentation site reorganisation
 *   java
+    *   [#784](https://github.com/pmd/pmd/issues/784): \[java] ForLoopCanBeForeach false-positive
```

Comment:
this change should be under `java-bestpractices` as the rule is in that category